spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
```

Comment:
Perfect implementation on solving the N+1 problem. However, I'm facing errors with your testing:

What I run:
1. bundle
2. rails db:create
3. rspec

then...

<img width="844" alt="Screenshot 2023-01-31 at 21 25 02" src="https://user-images.githubusercontent.com/69011963/215820361-2473728f-11c8-4ed7-98f6-f2b91486df35.png">

- [ ] Could you leave your instruction on what you are following up to test? Please make sure all your tests are working as expected! Thank you!

Reply:
Hi @Rustamxon7, yesterday we sent the same project with my partner and he got approved. What you have to do is add some information to the user, post, comment and like, instead you won't have information for the view

Reply:
I just created a better explanation to run the tests

Reply:
Hi, @Rustamxon7 I think you qualify **wrong**, Since this test only appears in the database of the person that is using it, we are using Postgres so we can upload the information that has Postgres on it, also, my peer will do a video to validate that the testing is correct.